frontend/src/pages/Setup/SetupPage.tsx

```diff
@@ -43,44 +36,101 @@ interface SetupStep {
 	tooltip?: string
 }
 
-enum PlatformType {
-	Html = 'Other',
-	React = 'React',
-	Vue = 'Vue.js',
-	NextJs = 'Next.js',
-	SvelteKit = 'SvelteKit',
-	Gatsby = 'Gatsby.js',
-	Angular = 'Angular',
+// The keys of these *_OPTIONS variables must match what we get back from the
+// quickstart endpoint in the docs.
+const PLATFORM_OPTIONS = {
+	client: 'Client',
+	server: 'Server',
+	other: 'Other',
+} as const
+
+const CLIENT_FRAMEWORK_OPTIONS = {
+	angular: 'Angular',
+	react: 'React',
+	next: 'Next',
+	vue: 'Vue',
+	gatsby: 'Gatsby',
+	other: 'Other',
 }
 
-enum BackendPlatformType {
-	Express = 'Express',
-	NextJs = 'Next.js',
-	Go = 'Go',
+const OTHER_OPTIONS = {
+	'self-host': 'Self Host',
+	'dev-deploy': 'Dev Deploy',
+}
+
+const BACKEND_LANGUAGE_OPTIONS = {
+	python: 'Python',
+	go: 'Go',
+	js: 'JavaScript',
+} as const
+
+const BACKEND_FRAMEWORK_OPTIONS = {
+	flask: 'Flask',
+	django: 'Django',
+	fastapi: 'FastAPI',
+	other: 'Other',
+	'aws-lambda': 'AWS Lambda',
+	'azure-functions': 'Azure Functions',
+	'google-cloud-functions': 'Google Cloud Functions',
+	gqlgen: 'GQLGen',
+	fiber: 'Fiber',
+	chi: 'Chi',
+	mux: 'Mux',
+	gin: 'Gin',
+	apollo: 'Apollo',
+	cloudflare: 'Cloudflare',
+	express: 'Express',
+	firebase: 'Firebase',
+	nodejs: 'Node.js',
+	trpc: 'tRPC',
+} as const
```

Comment:
can leverage new key - https://github.com/highlight/highlight.io/pull/420

https://github.com/highlight/highlight.io/blob/e85c06640e5648b41f7525edf1544de34995d942/components/QuickstartContent/QuickstartContent.tsx#L40

Reply:
Updated in https://github.com/highlight/highlight/pull/4532/commits/ab6a840778605c25cf9ebb86c4c5a10a7b3fb703